deploy/chart/templates/0000_50_olm_00-namespace.yaml

```diff
@@ -2,9 +2,17 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ .Values.namespace }}
+  labels: 
+    pod-security.kubernetes.io/enforce: restricted
+    pod-security.kubernetes.io/enforce-version: latest
 
 ---
 apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ .Values.operator_namespace }}
+  labels:
+    pod-security.kubernetes.io/enforce: baseline
```

Comment:
@anik120 @perdasilva any pod installed on this namespace will NOT have ONLY restricted-v2 SCC. 
Therefore, if you do not inform the RunAsUser and you set RunAsNonRoot true the SCC will not inject the RunAsUser. 

It will fail in Openshift when the image does not define the UserID

Reply:
/hold